src/main/groovy/org/pih/warehouse/report/MultiLocationInventoryReportCommand.groovy

```diff
@@ -35,11 +35,11 @@ class MultiLocationInventoryReportCommand implements Validateable {
     }
 
     Boolean getIsActionRun() {
-        return actionButton?.equalsIgnoreCase("run")
+        return actionButton?.equalsIgnoreCase("run") ?: false
```

Comment:
@alannadolny by default all Grails properties are `nullable: false`. To make it work "in a clean way", you need to add the transients to the constraints and mark them as `nullable: true`.
So to add:
```groovy
static constraints = {
...
isActionRun(nullable: true)
isActionDownload(nullable: true)
}